src/server.js

```diff
@@ -11,6 +11,20 @@ const handle = app.getRequestHandler();
 app.prepare().then(() => {
   const server = express();
 
+  // Old urls from the classic rails app
+  server.get(
+    [/([a-zA-Z_-]*)\/map\/.+/, /([a-zA-Z_-]*)\/community_support\/new.*/, /([a-zA-Z_-]*)\/embed.*/],
```

Comment:
-1: Shouldn't all `[a-zA-Z_-]*` be `[a-zA-Z_-]+`?

Reply:
This is part of the regexp represents the locale, which is optional.

Reply:
I guess I am just nit-picky about undocumented regexp. But the extra slash is not optional - so it only matches eg. `/community_support/new` but not `community_support/new`. Either the whole first part is optional, then the slash should be optional. Or it is required, then it should need at least one character before the slash.